examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java

```diff
@@ -90,7 +90,9 @@ public static void main(String[] args) throws Exception {
     if (args != null && args.length > 0) {
       conf.setNumWorkers(3);
 
-      StormSubmitter.submitTopologyWithProgressBar(args[0], conf, builder.createTopology());
+      for (String name: args) {
```

Comment:
This was code that I put in to be able to reproduce the problem consistently.  If you want me to remove it I am happy to.  It does not really provide any real value beyond that.